xbmc/Application.cpp

```diff
@@ -1073,13 +1073,12 @@ void CApplication::CreateUserDirs() const
   CDirectory::Create("special://temp/");
   CDirectory::Create("special://logpath");
   CDirectory::Create("special://temp/temp"); // temp directory for python and dllGetTempPathA
-
```

Comment:
This empty line shouldn't be removed. Once done could you please squash all commits into a single one?